flink-table-store-core/src/main/java/org/apache/flink/table/store/file/operation/AbstractFileStoreScan.java

```diff
@@ -249,22 +229,14 @@ private boolean filterManifestFileMeta(ManifestFileMeta manifest) {
                         manifest.partitionStats().fields(partitionStatsConverter));
     }
 
-    private boolean filterManifestEntry(ManifestEntry entry) {
+    protected boolean filterManifestEntry(ManifestEntry entry) {
```

Comment:
I think a `filterByPartitionBucket` here. And add a abstract class `filterByStats` is better.